parser/prism/Translator.cc

```diff
@@ -283,7 +282,7 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
             // parameters. So we need to extract the args vector from the Args node, and insert the locals at the end of
             // it.
             auto sorbetArgsNode = translate(up_cast(paramsNode->parameters));
-            auto argsNode = parser::cast_node<parser::Args>(sorbetArgsNode.get());
+            auto argsNode = parser::NodeWithExpr::cast_node<parser::Args>(sorbetArgsNode.get());
```

Comment:
This was going to bite us sooner or later, so I'm changing them all up front. This file should not use the `parser::isa_node` or `parser::cast_node` directly, because they won't treat `NodeWithExpr` the way we want (by casting their wrapped node).